### ISSUE TITLE
give examples of pass trees and structure

### ISSUE THREAD

**Issue by brabalan** [passB/passB#33]
It would be most useful to add some examples for the Matcher and the FileFormat strategies. For instance, I currently store the url for a login as a `url:` line. Is it supported? Is there a way to use the same password for several urls? (I know it's bad, but this is because of the CAS of where I work for.)

**Comment by phryneas** [comment]
Hi,
currently the only supported mapping between url and entry is the file name, as everything else would require to decrypt every pass entry (which can take up to a few minutes on a big store).

So for your current use case I would recommend to add symlinks named like the other websites, all pointing to the same pass file.

Currently, passB only allows to show an entry, or fill the currently open web page. But I guess I'll add a "browse to page and then fill the form" option soon, in which case I'll be honoring something like a `url:` prefix for the login page.

**Comment by brabalan** [comment]
Good point about decrypting. I'll do it using the symlink approach. Thanks!

I realize I worded the issue poorly, putting too many things into it. I would still like examples of the matcher, filler, and so on.

**Comment by phryneas** [comment]
Yes, I almost missed that part. You're right about that - some of the features still need more explanation than they currently have :)
I'll leave this issue open so I hope I'll come around to it soon. But I'll also add a "first good issue label" - if you feel like writing something,  I would be very glad to get some help.

If you want to help & need to understand how the Matcher is currently working and what the options are doing, you can take a look at the test for the SimpleMatcher: https://github.com/passB/passB/blob/master/src/__tests__/PluggableStrategies/Matchers/SimpleMatcher/SimpleMatcher.test.ts
If you want to add a hint to the options page, you can simply add a translation label somewhere in https://github.com/passB/passB/blob/master/src/PluggableStrategies/Matchers/SimpleMatcher/OptionsPanel.tsx and a translation string in https://github.com/passB/passB/blob/master/extension/_locales/en/messages.json

Again: no pressure - I'll come around to it eventually, but if you think you're up to it, I'd be happy about a contribution :)
